README.md

```diff
@@ -9,21 +9,59 @@ server is deployed at:
 
 https://go-acd.appspot.com
 
-# Install
+# Usage
+
+In order to use this library, you must authenticate through the token server. A  (e.g. https://go-acd.appspot.com).
 
-This project is go-gettable:
+Get and build:
 
 ```
-go get gopkg.in/acd.v0/...
+$ go get gopkg.in/acd.v0/cmd/acd
+$ go build gopkg.in/acd.v0/cmd/acd
 ```
 
-# Usage
+Run:
+
+```
+$ $GOPATH/bin/acd ls acd://
```

Comment:
It's just an example. It has to make absolute sense to the person reading the README without us making any assumptions about their path.